ops/testing.py

```diff
@@ -1618,6 +1646,22 @@ def _storage_remove(self, storage_id: str):
         index = int(index)
         self._storage_list[name].pop(index, None)
 
+    def network_get(self, endpoint_name: str, relation_id: Optional[int] = None) -> '_NetworkDict':
```

Comment:
One of the reasons why we hesitated to add support for network-get mocking is that the network API is changing heavily with juju 3.0, and I am not sure that the change is 'done' yet. 
We wanted to avoid implementing the same thing twice, but I can see the reason for having support in older juju versions as well.

Point is, we should be careful to design this in such a way that it'll be forwards-compatible with the new juju3 network objects.

Can maybe @jameinel comment on the status of networking in juju, and whether you think this is a good moment to add network support into the harness?

Reply:
I think it's okay to work with what we have now, especially given that people are having to patch `_TestingModelBackend`.